style.css

```diff
@@ -52,6 +52,56 @@
     display: none;
   }
 
+  /* menu toggle styling */
+  .navbar-show {
+    position: absolute;
+    top: 0;
+    left: 0;
+    height: 100vh;
+    width: 100%;
+    background: #6070ff;
+    background-blend-mode: multiply;
+    margin: 0;
+    padding: 0;
+    opacity: 0.8;
+    display: flex;
+    flex-direction: column;
+    gap: 40px;
+    overflow: hidden;
+  }
```

Comment:
- [ OPTIONAL]  Your mobile nav does not have the blur in the figma. I suggest you make time and implement this feature. 🧑‍🎄